src/commands/generateProject.ts

```diff
@@ -84,14 +83,22 @@ export async function generateProject(clickedFileUri: vscode.Uri | undefined): P
       apiPackageName +
       " --model-package " +
       modelPackageName;
-    // run the generator command with a "progress" dialog
-    await vscode.window.withProgress(
-      {
-        location: vscode.ProgressLocation.Notification,
-        title: "Generating the MicroProfile Rest Client interface template...",
-      },
-      () => processUtil.exec(jarCommand)
-    );
+    try {
+      await generateRestClient(jarCommand);
+    } catch (e) {
+      if (e.message.includes(SPEC_VALIDATION_EXCEPTION)) {
+        const selection = await vscode.window.showInformationMessage(
+          "The yaml file provided failed the OpenAPI Generator specification validation. Would you like to generate without specification validation?",
```

Comment:
`The yaml file provided failed` -> `The provided yaml file failed`

This message also shows if a URL was provided instead of a file, but I can't think of a way to make the message more generic for that case.

Reply:
How about `The provided yaml failed`

Reply:
That's fine